scania-digital-design-system/sdds
# feat(code example): remove code example component and depedencies

Code example component was removed, and all it's depedencies from package.json.

**Solving issue**  
_In case of GitHub issue add # plus the number of the issue (for example #123) OR if it is Azure then AB# and number of ticket_
Fixes: #2471

**How to test**  
Check out the repo and make sure the code component (and it's deps. like higlight.js are removed). Might need an npm install for this.
